<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageStageEncoderWgtClass</class>
 <widget class="QWidget" name="PageStageEncoderWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1">
    <widget class="QComboBox" name="encSetDifferentialOutput">
     <item>
      <property name="text">
       <string>Differential</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Single ended</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Output:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="FloatSpinBox" name="maxOperatingFrequencySpin">
     <property name="suffix">
      <string> kHz</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Max. current consumption:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Manufacturer:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="varlengthLineEdit" name="manufacturerEdit">
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Part number:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="varlengthLineEdit" name="partnumberEdit">
     <property name="maxLength">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Max. operating frequency:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Supply voltage:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="FloatSpinBox" name="maxCurrentConsumptionSpin">
     <property name="suffix">
      <string> mA</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Pulses per revolution:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="PPRSpin"/>
   </item>
   <item row="4" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="FloatSpinBox" name="supplyVoltageMinSpin">
        <property name="suffix">
         <string> V</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string> - </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="FloatSpinBox" name="supplyVoltageMaxSpin">
        <property name="suffix">
         <string> V</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Output type:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="encSetPushpullOutput">
     <item>
      <property name="text">
       <string>Push-pull</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Open drain</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="encSetIndexchannelPresent">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Index channel:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="encSetRevolutionsensorPresent">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Revolution sensor:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="encSetRevolutionsensorActiveHigh">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Rev.sensor active is high:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>manufacturerEdit</tabstop>
  <tabstop>partnumberEdit</tabstop>
  <tabstop>maxOperatingFrequencySpin</tabstop>
  <tabstop>supplyVoltageMinSpin</tabstop>
  <tabstop>supplyVoltageMaxSpin</tabstop>
  <tabstop>maxCurrentConsumptionSpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
